R/python.R

```diff
@@ -760,6 +783,8 @@ py_set_attr <- function(x, name, value) {
 #' @export
 py_list_attributes <- function(x) {
   ensure_python_initialized()
+  if (py_is_module_proxy(x))
+    py_resolve_module_proxy(x)
```

Comment:
Looks like you're right. Should I take out the `ensure_python_initialized()` check? Would it ever be possible that we could get a non-module Python object before Python itself is initialized? (seems not possible but want to be sure since I haven't spent as much time in this codebase)